basic_portfolio/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Portfolio Page</title>
+<style>
+h1{
+	background-color: lavender;
+	height: 20%;
```

Comment:
So % is one unit of measure, but there are many others.  So when I say something like "% of the viewport" I'm actually referring to units of vh and vw, which are a % of the viewport.  If you use straight %, then it makes something a percent of the size of its parent.  Check them out <a href="http://www.w3schools.com/cssref/css_units.asp" target="_blank">here</a>